extensions/terminal-suggest/src/fig/execute.ts

```diff
@@ -21,7 +21,6 @@ export const executeCommandTimeout = async (
 	const command = [input.command, ...input.args].join(' ');
 	try {
 		console.info(`About to run shell command '${command}'`);
```

Comment:
Should we pass through the execute externals (rename to be more general) and include the log service? That way we can keep this on the debug log level.